pom.xml

```diff
@@ -308,6 +308,34 @@ limitations under the License.
           </excludes>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <version>3.1.2</version>
+        <configuration>
+            <configLocation>.mvn/checkstyle.xml</configLocation>
+            <encoding>UTF-8</encoding>
+            <consoleOutput>true</consoleOutput>
+            <failsOnError>true</failsOnError>
```

Comment:
Do we want this? Reading the docs it looks similar to `failsOnViolation`.

Reply:
`failsOnError ` can be removed yes, like `<property name="severity" value="warning"/>`